src/pages/ReportView/WorkloadSummary/WorkloadSummary.tsx

```diff
@@ -188,7 +188,7 @@ class WorkloadMigrationSummary extends React.Component<Props, State> {
                             { formatPercentage(percentages[2], 0) } RHEL
                         </h2>
                         <h3 className="pf-c-title pf-m-1xl">
-                            Workloads running, or possible to migrate to Red Hat Enterprise Linux
+                            Workloads possible to migrate to RHEL
```

Comment:
i figured out that for consistency we should now use the acronym in this label.
```suggestion
                            Workloads possible to migrate to Red Hat Enterprise Linux
```